protected/humhub/modules/admin/views/authentication/authentication.php

```diff
@@ -39,6 +39,7 @@
         <?= $form->field($model, 'blockUsers')->checkbox() ?>
         <?= $form->field($model, 'hideOnlineStatus')->checkbox() ?>
         <?= $form->field($model, 'allowUserTopics')->checkbox(['data' => ['action-change' => 'admin.space.restrictTopicCreation']]) ?>
+        <?= $form->field($model, 'disableChoicesIos')->checkbox() ?>
```

Comment:
@marc-farre Do we still need this option? At least we should move it somewhere into the config file?

Reply:
Yes, we do need it, until we provide Apple authentication.
But I agree, we could move it to the config file.

Reply:
@luke- Should we put new config var in the `params/humhub` [here](https://github.com/humhub/humhub/blob/master/protected/humhub/config/common.php#L270), or maybe create some new block like `params/mobile`?

Please note previous value from the module will be lost, or do you know soma way to migrate a setting value from DB to config file?

Reply:
@yurabakhtin No need to migrate, because I know the HumHub instance that is using it (it's for white label mobile app only).

Reply:
> Yes, we do need it, until we provide Apple authentication. But I agree, we could move it to the config file.

If it helps HumHub you can always improve @GreenMeteor's OAuth clients as needed as we do provide a number of different ones;
https://github.com/GreenMeteor/humhub-oauth/blob/master/oauth/Apple/Apple.php

Reply:
@ArchBlood very interesting. Have you successfully tested it?

Reply:
> A simple param in the humhub block is fine for now.

@luke- Commit https://github.com/humhub/humhub/pull/7509/commits/9bbfd7c210f84ad360a686610a0ad42bf1421ad4.